src/main/java/io/citrine/jcc/search/dataset/result/DatasetMultiSearchResultElement.java

```diff
@@ -0,0 +1,23 @@
+package io.citrine.jcc.search.dataset.result;
+
+import io.citrine.jcc.search.core.result.BaseMultiSearchResultElement;
+
+/**
+ * Class to store the information in a single search result as a part of a dataset multi-query.
+ *
+ * @author Kyle Michel
+ */
+public class DatasetMultiSearchResultElement extends BaseMultiSearchResultElement<DatasetSearchResult> {
```

Comment:
I might be hung up on naming. To me saying BaseMultiSearchResultElement implies that it would have implemented subclasses that you would actually be using. If we are saying that there is _one_ MultiSearchResultElement, then that is clearer. I will push that out.